Sources/SwiftDocCUtilities/Action/Actions/PreviewAction.swift

```diff
@@ -59,8 +59,6 @@ public final class PreviewAction: Action, RecreatingContext {
     
     /// A unique ID to access the action's preview server.
     let serverIdentifier = ProcessInfo.processInfo.globallyUniqueString
-
-    private let diagnosticEngine: DiagnosticEngine
```

Comment:
One bug was that this engine didn't have any consumers, so there was nothing to write the diagnostics that it received.